src/main/java/com/powsybl/openloadflow/sensi/AbstractSensitivityAnalysis.java

```diff
@@ -129,7 +134,9 @@ protected static Terminal getEquipmentRegulatingTerminal(Network network, String
 
         boolean areVariableAndFunctionDisconnected(GraphDecrementalConnectivity<LfBus> connectivity);
 
-        boolean isConnectedToComponent(Set<LfBus> connectedComponent);
+        boolean isVariableConnectedToSlackComponent(Set<LfBus> connectedComponent);
```

Comment:
What is for me a bit hard to understand if that these two functions are not used for pre-contingency state. It is an incoherent behavior, no?

Reply:
Those two functions are not used for pre-contingency state indeed and we only check if variable and function element are in slack connected component. This may result to an incoherent behavior regarding SKIP status as discussed in another comment.